docs/configuration/defaults.md

```diff
@@ -96,32 +96,6 @@ The preprocessing parameters that each data type accepts can be found in [dataty
 Note that different features with the same datatype may require different preprocessing. Type-global preprocessing works
 in tandem with feature-specific preprocessing configuration parameters, however, feature-specific configurations override the global settings.
 
```

Comment:
Maybe keep this section, but change `word_length_limit` to `max_sequence_length` and change `most_common_words` to `most_common`?